### ISSUE TITLE
Split the data into a training set and test set

### ISSUE THREAD

**Issue by jo-elimu** [elimu-ai/ml-storybook-reading-level#3]
Split the previously imported dataset (#1) into a training set and test set, using the `reading_level` column as the predictor.

Then write the code for training the machine learning model.

---

- Depends on #2
